payments/mobile-pay/other-features.md

```diff
@@ -549,7 +549,7 @@ between the captured amount and the authorized amount.
 
 ```mermaid
 sequenceDiagram
-  Merchant->>PayEx: POST <creditcard cancellation>
+  Merchant->>PayEx: POST <mobilepay cancellation>
```

Comment:
```suggestion
  Merchant->>PayEx: POST <MobilePay Online cancellation>
```

Reply:
Since the `cancellation` applies directly to the API, which is called `mobilepay`, I don't agree with this change.

Reply:
Since the `cancellation` applies directly to the API, which is called `mobilepay`, I don't agree with this change.

Reply:
Or perhaps I do. I'm not sure.

Reply:
I was a bit uncertain regarding these ones as well. I left them out for the same reason that Asbjørn is hesitating :)

In the words of Stewie Griffin: "Do I leave it in or do I take it out!"